### ISSUE TITLE
Correlation/coherence between allographs of different graphemes

### ISSUE THREAD

**Issue by simonastoyanova** [kingsdigitallab/crossreads#67]
Two levels of this sort of comparison across graphemes, one already possible in Archetype and the annotation tool. Going by the facet component-feature we can see, for example, all instances of apex+serif, whether it appears in A, V, M, Λ or whichever other grapheme. 

Deeper level of comparison which isn't yet handled is: does this allograph of A appear often in the same inscriptions as this allograph of P? This would allow the identification of styles (sometimes called scripts, cf. insular) across the alphabet rather than focusing on the variants of a single allograph or component. 

Two distinct styles in ISic000093, though they are kept separate in the layout: https://kingsdigitallab.github.io/crossreads/annotator.html?obj=http://sicily.classics.ox.ac.uk/inscription/ISic000093&img=ISic000093.jpg&scr=latin-1 
The Fs and the Ps have long and curved setifs on the downstroke, to the point of being consufind (F or E). How often does that happen and it is also co-occuring with the curvy sefir on the R right bar or the N left stroke? 

This will become even more relevant for later texts where some uncial and cursive forms start to creep in from other genres/materials into stone inscriptions. Lunate epsilon, sigma, uncial omega etc.

**Comment by simonastoyanova** [comment]
Follow up with some details.

[ISic002745](http://sicily.classics.ox.ac.uk/inscription/ISic002745) has no archaeological context and no date that spans to less than three centuries. 

![Screenshot 2025-01-13 at 16 30 08](https://github.com/user-attachments/assets/4d1d0ab7-e932-4c76-99f0-b6f1bb31dec5)

To pinpoint something more precise, we look at where/when else these letterforms appear together, e.g. this type of alpha and this type of omega here, in comparison to the other types in the corpus:

![Screenshot 2025-01-13 at 16 30 24](https://github.com/user-attachments/assets/eba7a776-f528-46f2-8299-4160cbafe8fc)
![Screenshot 2025-01-13 at 16 30 31](https://github.com/user-attachments/assets/2c72cfdc-ad12-4ff4-8e63-a5df77716e34)

We find some other inscriptions with their dates and places which also have these letterforms appearing together:

![Screenshot 2025-01-13 at 16 30 40](https://github.com/user-attachments/assets/6c384bd0-2726-478d-bfc3-901edad520eb)

Searching for co-occurances of types across the corpus will be extremely useful both for internal research and visualisation and dissemination.

**Comment by simonastoyanova** [comment]
@geoffroy-noel-ddh I've added a few slides and an better use case in this issue which was already open but somewhat vague.

**Comment by geoffroy-noel-ddh** [comment]
Thank you very much, that's very useful. Neil and I will think of ways we can discover and display correlations and discuss with you. If you have any specific idea about how to go about it with a user interface and/or a workflow/method using the existing data, feel free to share with us.

**Comment by simonastoyanova** [comment]
One thing that comes to mind, which we briefly discussed, is a potential export of a search or comparison. Or even simpler - a stable uri for a search which can be shared with other people. Example: this week I wanted to send Jon a few comparisons to look at and use in a presentation. Pasting the uri in a new window gives a blank page - I assume because each comparison is done on the fly and doesn't retain the information? So instead a sent him screenshots. Let me know what you think.

**Comment by simonastoyanova** [comment]
@geoffroy-noel-ddh yes, it was alpha types 1.2 and 2. Here is the screenshot with the hover with a l ist of inscriptions which don't have -600 but are clustered in a graph with -600:

<img width="1727" height="845" alt="Image" src="https://github.com/user-attachments/assets/e23fd284-2687-410d-be44-0096c00d580e" />

**Comment by geoffroy-noel-ddh** [comment]
@simonastoyanova I can't reproduce those results. I get much fewer inscriptions for those two queries. Can you delete the queries and recreate them to see if it's still there?

<img width="918" height="520" alt="Image" src="https://github.com/user-attachments/assets/6b0df9b6-6734-44b6-9349-c041769b3827" />
